crates/symbolicator-service/src/services/download/mod.rs

```diff
@@ -322,12 +322,14 @@ impl DownloadService {
             Ok(Ok(res)) => res,
         };
 
-        if matches!(
-            result,
-            Err(CacheError::DownloadError(_) | CacheError::Timeout(_))
-        ) {
+        if let Err(ref e @ (CacheError::DownloadError(_) | CacheError::Timeout(_))) = result {
             metric!(counter("service.download.failure") += 1, "source" => &source_metric_key);
 
+            if source_metric_key == "sentry:project" {
+                ::sentry::configure_scope(|scope| scope.set_extra("host", host.clone().into()));
```

Comment:
I guess that will be `sentry.internal`, so not too useful, but why not… If you set it as a `tag`, you can also filter and aggregate it

Reply:
`host` should be the `url` of the remote file for a Sentry source, or am I misunderstanding?

Reply:
yes exactly, I mean that the hostname internally looks something like that, and it is always the same across all `sentry:project` downloads, so it provides no value as a custom tag/extra.